TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/PersonDetail/Induction.cshtml

```diff
@@ -0,0 +1,70 @@
+@page "/persons/{personId}/induction"
+@using TeachingRecordSystem.SupportUi.Pages.Common;
+@model TeachingRecordSystem.SupportUi.Pages.Persons.PersonDetail.InductionModel;
+@{
+    Layout = "Layout";
+    ViewBag.SelectedTab = PersonSubNavigationTab.Induction;
+    var person = HttpContext.Features.GetRequiredFeature<CurrentPersonFeature>();
+}
+
+@section BeforeContent {
+    <govuk-back-link href="@LinkGenerator.PersonDetail(person.PersonId)" data-testid="induction-backlink">Back to record</govuk-back-link>
+}
+@if (!String.IsNullOrEmpty(Model.StatusWarningMessage))
+{
+    <govuk-warning-text icon-fallback-text="Warning" data-testid="induction-status-warning">@Model.StatusWarningMessage</govuk-warning-text>
+}
+
+@if (Model.Status != InductionStatus.None)
+{
+    <govuk-summary-card data-testid="induction-card">
+        <govuk-summary-card-title data-testid="induction-title">Induction details</govuk-summary-card-title>
+        <govuk-summary-list>
+            <govuk-summary-list-row data-testid="induction-status">
+                <govuk-summary-list-row-key>
+                    Induction status
+                </govuk-summary-list-row-key>
+                <govuk-summary-list-row-value>
+                    @Model.StatusString
+                </govuk-summary-list-row-value>
+            </govuk-summary-list-row>
+
+            @if (Model.ExemptionReasons != null && Model.ExemptionReasons.Any())
+            {
+                <govuk-summary-list-row data-testid="induction-exemption-reasons">
+                    <govuk-summary-list-row-key>
+                        Exemption reason
+                    </govuk-summary-list-row-key>
+                    <govuk-summary-list-row-value>
+                        @Model.ExemptionReasonsText
+                    </govuk-summary-list-row-value>
+                </govuk-summary-list-row>
+            }
+
+            @if (Model.StartDate != null) // CML logic should be based on status?
+            {
+                <govuk-summary-list-row data-testid="induction-start-date">
+                    <govuk-summary-list-row-key>
+                        Induction start date
+                    </govuk-summary-list-row-key>
+                    <govuk-summary-list-row-value>
+                        @Model.StartDate?.ToString(UiDefaults.DateOnlyDisplayFormat)
+                    </govuk-summary-list-row-value>
+                </govuk-summary-list-row>
+            }
+
+            @if (Model.CompletionDate != null) //CML logic should be based on status?
```

Comment:
ticket specifies statuses where start date shouldn't show but I'd put the logic in to show it if it's there - should I change it? (same Q for CompletionDate)

Reply:
I think Status might be better; we may get some bad migrated data from DQT that doesn't have a date even though the status says it should. In such a case I think we'd want to see the row (albeit with an empty value).

Reply:
Don't know how to test for that case at the mo - when the status says show start date but there's no start date - The TestData builder sems to add a default start date

Reply:
done